docs/administration/crud-controller/getting-started/configure-list-page.md

```diff
@@ -1 +1,117 @@
-## Configure List Page
\ No newline at end of file
+## Configure List Page
```

Comment:
the documentation is somehow weirdly structured. Link `Getting started` point to `Configure list page`

Reply:
fixed